cmd/package.go

```diff
@@ -269,10 +279,8 @@ func CollectPackage(repo *util.Repo, packageDir string, pullMissing bool, custom
 		pkg.Require = append(genRuntime.GetDependencies(), pkg.Require...)
 	}
 
-	// The bootstrap package is implicitly required by every application package,
-	// so we add it to the list of required packages. Even if user has added
-	// the bootstrap manually, this will not result in overhead.
-	pkg.Require = append(pkg.Require, "osv.bootstrap")
+	// Prepend implicitly required packages.
+	pkg.Require = append(implicitRequire, pkg.Require...)
```

Comment:
We are making the implicit require first giving other packages ability to override bootstrap?